.github/workflows/add-to-whats-new.yml

```diff
@@ -0,0 +1,16 @@
+name: Add comment about adding a What's new note
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  add-comment:
+    if: ${{ ! github.event.pull_request.head.repo.fork && contains(github.event.pull_request.labels.*.name, 'Add to what''s new') }}
```

Comment:
isn't the label value lowercase?

Reply:
Argh forget it, github will make it lowecase automatically anyway 

> This function is not case sensitive
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#contains